src/parser/xasl_generation.c

```diff
@@ -21798,6 +21799,11 @@ parser_generate_xasl (PARSER_CONTEXT * parser, PT_NODE * node)
   node->next = NULL;
   parser->dbval_cnt = 0;
 
+  {
```

Comment:
Is there any reason this code block is enclosed in braces?

Reply:
Declare and call extern functions that are only used here. I was trying to wrap it up with blocks to make it more visible.